content/tutorials/get-started/get-started-using-php.md

```diff
@@ -0,0 +1,288 @@
+```
+
+The result returns a `Wallet` instance:
+
+```php
+// print wallet properties
+print_r([
+  'publicKey' => $wallet->getPublicKey(),
+  'privateKey' => $wallet->getPrivateKey(),
+  'classicAddress' => $wallet->getAddress(),
+  'seed' => $wallet->getSeed()
+]);
+
+// output
+// Array
+// (
+//    [publicKey] => ED2C4CE69F663254840905AEF5FB8596FC243EDEBE0295A6ECEE86CE8EB8F76210
+//    [privateKey] => -HIDDEN-
+//    [classicAddress] => rBi9u1P3ofRKTFPFTgrguANz2wRqsdKHvm
+//    [seed] => -HIDDEN-
+//)
+
+```
+
+For testing and development purposes, you can use the `fundWallet()` helper function on the XRP Ledger [Testnet](parallel-networks.html):
+
+```php
+<?php
+
+// Use the Composer autoloader
+require __DIR__ . '/vendor/autoload.php';
+
+// Imports
+use XRPL_PHP\Client\JsonRpcClient;
+use XRPL_PHP\Wallet\Wallet;
+use function XRPL_PHP\Sugar\fundWallet;
+
+// Create a client using the Testnet
+$client = new JsonRpcClient("https://s.altnet.rippletest.net:51234");
+
+// Create a new wallet
+$wallet = Wallet::generate();
+
+// Fund (and activate) the wallet
+$fundWalletResponse = fundWallet($client, $wallet);
+```
+
+### {{n.next()}}. Query the XRP Ledger
+
+You can query the XRP Ledger to get information about [a specific account](account-methods.html), [a specific transaction](tx.html), the state of a [current or a historical ledger](ledger-methods.html), and [the XRP Ledger's decentralized exchange](path-and-order-book-methods.html). You need to make these queries, among other reasons, to look up account info to follow best practices for [reliable transaction submission](reliable-transaction-submission.html).
+
+Here, we'll use the [`JsonRpcClient` we constructed](#1-connect-to-the-xrp-ledger) to look up information about the [account we got](#2-get-account) in the previous step.
+
+```php
+<?php
+
+// Use the Composer autoloader
+require __DIR__ . '/vendor/autoload.php';
+
+// Imports
+use XRPL_PHP\Client\JsonRpcClient;
+use XRPL_PHP\Wallet\Wallet;
+use function XRPL_PHP\Sugar\fundWallet;
+
+// Create a client using the Testnet
+$client = new JsonRpcClient("https://s.altnet.rippletest.net:51234");
+
+// Create a new wallet
+$wallet = Wallet::generate();
+
+// Fund (and activate) the wallet
+$fundWalletResponse = fundWallet($client, $wallet);
+
+// Create an AccountInfo request "method"
+$accountInfoRequest = new AccountInfoRequest(
+    account: $wallet->getAddress(),
+    ledgerIndex: 'validated'
+ );
+
+// Send AccountInfo request
+$accountInfoResponse = $client->syncRequest(($accountInfoRequest));
+
+print_r($accountInfoResponse);
+```
+
+### {{n.next()}}. Starting the script
+
+Now, we have a simple application that:
+
+1. Creates an account on the Testnet.
+2. Connects to the XRP Ledger.
+3. Looks up and prints information about the account you created.
+
+To run the app, you can copy the `_code-samples/get-started/php` directory [Github](https://github.com/XRPLF/xrpl4j-sample) and run it from the command line:
```

Comment:
The link goes to the xrpl4j example again—xrpl4j is a bit of an exception since its "basic" setup is complicated enough that we set up an example project repo for it. For the other languages, you can just link to the source repo. If you use the `target.github_branch` and `target.github_forkurl` variables, the link will work correctly in the PR test build even before it's merged upstream.

```suggestion
To run the app, you can copy the code from [this website's GitHub Repository]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/get-started/php/) and run it from the command line:
```